### ISSUE TITLE
General Question regarding development status

### ISSUE THREAD

**Issue by gjduran** [OCA/management-system#242]
Hi,
Recently I'm again investigating the functionality of the management system which I remembered from a long time ago (OpenERP 5??). I noticed quit some odd behaviours compared to what I remember from the early days. i.e. Provided templates doesn't seem to load. General settings does not respond to tick boxes. As suggested I'm looking at: http://3354556-9-0-6c3b5a.runbot2.odoo-community.org/web?debug=1#view_type=form&model=mgmtsystem.config.settings&menu_id=188&action=213 and the following settings are active.
 Management Systems
Systems
Quality (ISO 9001) is enabled
Environment (ISO 14001) is disabled
Hygiene & Safety (OHSAS 18001) is disabled
Information Security (ISO 27001) is disabled
Applications
Applications
Actions (immediate, corrective, preventive) and improvement opportunities  is enabled
Nonconformities  is enabled
Claims  is enabled
Audits  is enabled
Top management reviews  is enabled
Manuals
Manuals
	
Quality Manual template based on the ISO 9001:2008 standard   is disabled
Environment Manual template based on the ISO 14001:2004 standard  is disabled
Health & Safety Manual template based on the OHSAS 18001 standard  is disabled
Information Security Manual template based on ISO 27001  is disabled
Documentation
Documentation
	
Procedures is disabled
Environmental Aspects is disabled
Hazards is enabled
Feared Events  is disabled
Neither of the disabled functions can be switched to enabled, after applying these just remain disabled.
The same I'm experiencing on my local setup which made me to further investigate what I was doing wrong.
Basically I find this management system very interesting but wonder what may be expected from it's own quality.
Any response will be highly appreciated.

**Comment by pedrobaeza** [comment]
What do you mean by disabled? Which version? Probably modules won't be migrated yet.

**Comment by gjduran** [comment]
If you go to the settings page on the runbot instance and try to enable either of the disabled options nothing happens. Somehow templates won't load though they are part of version 11 tree.

**Comment by pedrobaeza** [comment]
As said, they are not migrated. Follow them on https://github.com/OCA/management-system/issues/200

P.S.: I have never liked to put all the features in the main module if they are not migrated. cc @max3903

**Comment by gjduran** [comment]
Thanks for responding so rapidly. I will follow on #200.
